desktop.css

```diff
@@ -0,0 +1,330 @@
+  }
+
+  .portfolio-image {
+    order: 0;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    margin-left: 0.2rem;
+    width: 100%;
+  }
+
+  .portfolio-head {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-bottom: 0;
+  }
+
+  .framed {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    position: static;
+    width: 297px;
+    height: 40px;
+    justify-content: space-between;
+  }
+
+  .canopy {
+    position: static;
+    width: auto;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .backend {
+    position: static;
+    width: auto;
+    height: 24px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .year {
+    position: static;
+    width: auto;
+    height: 24px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6a768a;
+  }
+
+  .lang2 {
+    display: flex;
+    flex-direction: row;
+    float: left;
+    gap: 1rem;
+  }
+
+  .a-lang {
+    color: #6070ff;
+    text-decoration: none;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    background: #ebebff;
+    border-radius: 8px;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    margin: 0 0.3rem;
+  }
+
+  .aboutus-card {
+    display: flex;
+    width: 70%;
+    gap: 5%;
+  }
+
+  .aboutus-content {
+    order: 0;
+    width: 38%;
+  }
+
+  .marginleft18 {
+    margin-left: 18.05%;
+  }
+
+  .frame2 {
+    width: 55%;
+    align-self: flex-start;
+    padding-top: 2rem;
+  }
+
+  .frame3 {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .l-list2 {
+    display: flex;
+  }
+
+  .contact-container {
+    width: 100%;
+    background-color: #6070ff;
+    border-radius: 100px 0 0 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url("./src/contactformbackgroundd.png");
+    background-position: center bottom;
+    background-size: 90% 85%;
+    background-repeat: no-repeat;
+  }
+
+  .contact-text {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    color: white;
+    margin-top: 10rem;
+    margin-bottom: 0;
+  }
+
+  .contact-paragraph {
+    width: 30%;
+    text-align: center;
+    color: #ebebff;
+    margin-bottom: 1rem;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    width: 25%;
+    gap: 1rem;
+    align-items: center;
+  }
+
+  form input {
+    width: 100%;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    color: #172b4d;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 20px;
+    height: 20px;
+    margin-bottom: 18px;
+    padding-left: 10px;
+  }
```

Comment:
The form's input fields style doesn't match those of the template design. Please review the Figma template and fix them.